packages/flame/lib/src/components/component.dart

```diff
@@ -131,34 +131,28 @@ class Component with Loadable {
     children.forEach((c) => c.update(dt));
   }
 
-  @mustCallSuper
-  void render(Canvas canvas) {
-    preRender(canvas);
-  }
+  void render(Canvas canvas) {}
 
-  @mustCallSuper
   void renderTree(Canvas canvas) {
+    preRender(canvas);
```

Comment:
we should probably add comments to preRender to let users know that they are responsible for saving/restoring the canvas

Reply:
ok, I added a mention about that in the doc-comment of `preRender()`.